### ISSUE TITLE
Check for unused objects in encapsulated package

### ISSUE THREAD

**Issue by nununo** [larshp/abapOpenChecks#334]
For a given package, it would be great if I could find a list of all objects which:

- are not used by any other object in the same package
- are not part of a package interface

Which means they're pretty much useless and probably obsolete.

**Comment by larshp** [comment]
yep, would probably do this as a separate program, as DDIC objects are not handled very well by code inspector
